src/compas_view2/app/config_default.json

```diff
@@ -14,8 +14,17 @@
     "view": {
         "show_grid": true,
         "view_mode": "shaded",
+        "view_port": "PERSPECTIVE",
```

Comment:
personally, i would not add an underscore in keys like these. you could consider "viewmode" and "viewport" as one word. "show_grid" is different in that sense...

Reply:
Yes. I didn't find the naming conversions for the configuration / .jason file so I named it this way. All the items in this file, including some long words, like `enable_propertyform`, `flow_view_size`, `zoom_selected`, `view_perspective`, `background_color`, and `selection_color`, then need to be changed. Shall we decide fixing them all?

We could create a separate PR/Issue for this